Source/Tests/skip_tests.lst

```diff
@@ -0,0 +1 @@
+test_DefaultTemplateMeasurement_GetUiDetails_CorrectUiPathUrl.vi
```

Comment:
This test doesn't run because it requires measurementlink-labview to be installed. Our test runner currently doesn't install measurementlink-labview and instead runs the tests against source.